src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,68 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.lang.reflect.Array;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private K[] keys;
+    private V[] values;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

0 is default value for int